packages/react-charts/src/components/ChartTheme/ChartThemeTypes.ts

```diff
@@ -31,97 +8,8 @@ export const ChartAxisTheme: ChartThemeDefinition = AxisTheme;
 export const ChartBaseTheme: ChartThemeDefinition = BaseTheme;
 
 /**
- * Bullet comparative error measure theme
- * @private
- */
-export const ChartBulletComparativeErrorMeasureTheme: ChartThemeDefinition = BulletComparativeErrorMeasureTheme;
```

Comment:
Would you classify these as breaking changes? Could products be importing these themes you've deleted?

Reply:
Marked deprecated